kafka/signed_token_redeem_handler.go

```diff
@@ -200,14 +211,20 @@ func SignedTokenRedeemHandler(
 	var resultSetBuffer bytes.Buffer
 	err = resultSet.Serialize(&resultSetBuffer)
 	if err != nil {
-		return fmt.Errorf("request %s: failed to serialize result set: %w",
-			tokenRedeemRequestSet.Request_id, err)
+		message := fmt.Sprintf("request %s: failed to serialize result set", tokenRedeemRequestSet.Request_id)
+		return utils.ProcessingErrorFromErrorWithMessage(err, message, logger)
 	}
 
 	err = Emit(producer, resultSetBuffer.Bytes(), logger)
 	if err != nil {
-		return fmt.Errorf("request %s: failed to emit results to topic %s: %w",
-			tokenRedeemRequestSet.Request_id, producer.Topic, err)
+		message := fmt.Sprintf("request %s: failed to emit results to topic %s", tokenRedeemRequestSet.Request_id, producer.Topic)
```

Comment:
similarly to the above, I don't think I saw kafka errors in `ErrorIsTemporary` - but there probably should also be?